samples/excel/09-events/selection-changed.yaml

```diff
@@ -0,0 +1,116 @@
+name: Selection Changed
+description: Add and remove an event handler on the selection changed event
+host: EXCEL
+api_set: {}
+script:
+    content: |
+        $("#add-event-handler").click(addEventHandler);
+        $("#remove-last-event-handler").click(removeLastEventHandler);
+        $("#remove-all-event-handlers").click(removeAllEventHandlers);
+
+        let lastHandler: OfficeExtension.EventHandlerResult<Excel.Workbook>;
+        let workbook: Excel.Workbook;
+
+        async function addEventHandler() {
+            try {
+                if (workbook !== undefined) {
+                    await Excel.run(workbook, async (context) => {
+                        workbook = context.workbook;
+                        lastHandler = context.workbook.onSelectionChanged.add(onSelectionChanged)
+
+                        await context.sync();
+                    })
+                } else {
```

Comment:
An else to an "!==" if statement implies that the blocks of code should be reversed.